lib/src/facade.dart

```diff
@@ -1,12 +1,53 @@
-import 'filter_builder_base.dart';
-import 'operators.dart';
-import 'values.dart';
+import 'filter_builder/filter_builder_base.dart';
+import 'filter_builder/operators.dart';
+import 'filter_builder/values.dart';
+import 'utils/_exports.dart';
 
 /// Convenience class to access all Meilisearch filter expressions
 
 class Meili {
   Meili._();
 
+  /// Takes an input string that can have highlighting and returns its parts.
+  /// each part has an isHighlighted text
+  static Iterable<MeiliHighlightableStringPart> parseHighligtableString({

```

Comment:
What do you think about moving this method to meilisearch-flutter?

I don't know if it will be a use case for it from this package. 🤔

Reply:
we can definitely move it to the flutter package, but I thought it might be helpful for server scenarios, where users can't depend on flutter